pyrobuf/parse_proto.py

```diff
@@ -173,6 +194,22 @@ def tokenize(self, disabled_token_types):
             raise Exception("Unexpected character '{}' on line {}: '{}'".format(
                 self.string[pos], line + 1, self.lines[line]))
 
+    def _handleComment(self, token, previous_token):
+        if token.token_type == 'MESSAGE' or token.token_type == 'ENUM' :
+            token.comment = self._lastComment              
+        self._lastComment = None
+
+        if token.token_type != "COMMENT_OL" and token.token_type != "COMMENT_ML":            
```

Comment:
An `else` here would be cleaner.

Reply:
Or just check this first and return `False`.

Reply:
No, `_lastComment` has to be reset in every case.